### PR TITLE
people.rst: Fix grammar issues

### PR DESCRIPTION
**Pull request by mineo**

### CODE REVIEW

**Comment by lunaryorn** [comment]
Thank you very much.  

Please amend your commit message to remove the file name from it as this information is already in the diff.

File: doc/community/people.rst [hunk 1]
```diff
@@ -16,8 +16,8 @@ Maintainers
 * **Clément Pit--Claudel** (:gh:`cpitclaudel`, owner)
 
 We maintain Flycheck and all official extensions within the `Flycheck
-organisation`_, and set the direction and scope Flycheck.  We review and accept
```
Comment (lunaryorn) [inline doc/community/people.rst]: Please preserve double spacing at sentence end.

**Comment by mineo** [comment]
Done and force-pushed.

**Comment by cpitclaudel** [comment]
Thanks, LGTM. :)

**Comment by lunaryorn** [comment]
LGTM as well.

**Comment by lunaryorn** [comment]
@mineo Thank you very much. Merged 😊

**Pull request merged**